### ISSUE TITLE
matching fails unless content begins with space

### ISSUE THREAD

**Issue by jason-ha** [Stuk/eslint-plugin-header#26]
Using rule:
`'header/header': [2, 'line', 'my header comment.']`
with file:
```javascript
// my header comment.
console.log(1);
```
reports `incorrect header`. Changing rule to:
`'header/header': [2, 'line', ' my header comment.']`
will not report error and unblocks use.

In both rule cases `--fix` will produce:
```javascript
// my header comment.
console.log(1);
```
if error is detected. (Added line per issue #24 Autofix adds new line has been removed.)

In my setup, I am using `eslint 6.8.0`.

**Comment by tonyganchev** [comment]
This is the expected behavior. The rule strips only `//` with no empty space. This is to ensure that no arbitrary empty spaces are tolerated and all headers look exactly the same.
